sdk/communication/AzureCommunicationChat/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.2.0 (Unreleased)
+### New Features
```

Comment:
https://azure.github.io/azure-sdk/policies_releases.html#change-logs
This does not align with the changelog guidance. This should say "Features Added". Same for line 15.

Reply:
fixed them